plrustc/plrustc/src/main.rs

```diff
@@ -52,6 +51,8 @@ impl Callbacks for PlrustcCallbacks {
     }
 }
 
+// TODO: eventually we can replace this with:
+// rustc_driver::install_ice_hook("https://github.com/tcdi/plrust/issues/new", |_| ());
```

Comment:
When? :eyes:

Reply:
We actually don't use nightly >_>

Reply:
ah. when 1.71 lands on stable in a few weeks, then.